src/app/content-details/content-details.page.ts

```diff
@@ -308,6 +312,15 @@ export class ContentDetailsPage implements OnInit, OnDestroy {
     } else {
       this.generateTelemetry();
     }
+    this.telemetryGeneratorService.generatePageLoadedTelemetry(
```

Comment:
Generate this telemetry where the old impression telemetry is getting generated.